plugin/src/main/kotlin/com/ritense/plugin/PluginDeploymentListener.kt

```diff
@@ -29,23 +28,30 @@ import com.ritense.plugin.exception.PluginDefinitionNotDeployedException
 import com.ritense.plugin.repository.PluginActionDefinitionRepository
 import com.ritense.plugin.repository.PluginActionPropertyDefinitionRepository
 import com.ritense.plugin.repository.PluginDefinitionRepository
+import com.ritense.plugin.repository.PluginPropertyRepository
 import mu.KotlinLogging
 import org.springframework.boot.context.event.ApplicationStartedEvent
-import org.springframework.transaction.event.TransactionalEventListener
+import org.springframework.context.event.EventListener
 import java.lang.reflect.Field
 import java.lang.reflect.Method
 import java.lang.reflect.Parameter
+import com.ritense.plugin.annotation.PluginProperty as PluginPropertyAnnotation
 
 class PluginDeploymentListener(
     private val pluginDefinitionResolver: PluginDefinitionResolver,
     private val pluginDefinitionRepository: PluginDefinitionRepository,
+    private val pluginPropertyRepository: PluginPropertyRepository,
     private val pluginActionDefinitionRepository: PluginActionDefinitionRepository,
     private val pluginActionPropertyDefinitionRepository: PluginActionPropertyDefinitionRepository
 ) {
 
-    @TransactionalEventListener(ApplicationStartedEvent::class)
+    @EventListener(ApplicationStartedEvent::class)
```

Comment:
Wouldn't adding the transactional annotation to the test also work? That allows us to keep the annotation as-is here.

Reply:
After starting the application locally, I found that plugins were not deployed at all anymore.